### PR TITLE
Add notes from first weekly

### PR DESCRIPTION
**Pull request by friedeggs**

### CODE REVIEW

**Comment by jxnl** [comment]
this is great! lets use the naming convention you've set up in the future. I'll add a few comments to fill things out.

File: w17-0118-intro-to-data-science.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+##### data transforms:
+SQL, pandas, [dplyr](https://github.com/hadley/dplyr)
+
+- Sarth recommends [SQLZOO](http://sqlzoo.net/) for picking up SQL
+
+- Jason says `dplyr` is much nicer to use than `pandas`, and doesn't require memorizing as much documentation
+
+##### modelling:
+scikit-learn, glm (generalized linear model)
```
Comment (jxnl) [inline w17-0118-intro-to-data-science.md]: courses to learn more about this stuff would be 

STAT331 and STAT431 along with STAT444

File: w17-0118-intro-to-data-science.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+#### reporting:
+[ipython](https://ipython.org/),
+[rmarkdown](http://rmarkdown.rstudio.com/)
+- lets you run code and write text all on one page
+
+These are the best way for communicating your results, hands down. Always include an ipython notebook with your analysis, or write one up as a standalone!
+
+
+### other:
+[tidyverse](http://tidyverse.org/) is a set of R packages that work well together and make data analysis much smoother.
```
Comment (jxnl) [inline w17-0118-intro-to-data-science.md:49]: http://r4ds.had.co.nz/ is a great resource to learn everything tidyverse.

File: w17-0118-intro-to-data-science.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+# Cornerstones of data science
+
+There's a wealth of tools for data science. Where do you get started? Luckily for you, Jason Liu's narrowed it down to three core areas.
+Here are the things you should master:
+
+##### data transforms:
+SQL, pandas, [dplyr](https://github.com/hadley/dplyr)
+
+- Sarth recommends [SQLZOO](http://sqlzoo.net/) for picking up SQL
```
Comment (jxnl) [inline w17-0118-intro-to-data-science.md:15]: +1 got me through facebook interviews

File: w17-0118-intro-to-data-science.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
```
Comment (jxnl) [inline w17-0118-intro-to-data-science.md:16]: add a line for pandas.

i recommend https://www.youtube.com/watch?v=5JnMutdy6Fw as a source to learn it from scratch.

**Review by jxnl — COMMENTED** [review]
good shit thanks for writing it up. next time when i present I'll make sure that you don't have to start from scratch.

**Comment by friedeggs** [comment]
my bad, don't have write access to this repo. @jxnl can you merge?

thanks for all the feedback and tips!

**Pull request merged**